packages/navie/src/services/vector-terms-service.ts

```diff
@@ -136,6 +137,10 @@ export default class VectorTermsService {
       responseText = contentAfter(responseText, 'Terms:');
       responseText = trimFences(responseText);
       searchTermsObject = parseJSON(responseText) || parseText(responseText);
+      if (isEmpty(searchTermsObject)) {
```

Comment:
I think it's easier than this, because this case can only happen if the response is valid JSON (which we aren't actually asking for, but I suppose could happen). Otherwise, parseText is already just splitting the text on whitespace. Then there is a bunch of postprocessing that happens after this, e.g. to eliminate non-character words and very short words.